src/main.cpp

```diff
@@ -1077,14 +1193,16 @@ bool ContextualCheckTransaction(
         // Rules that apply generally before the next release epoch
     }
 
-    auto consensusBranchId = CurrentEpochBranchId(nHeight, consensus);
     auto prevConsensusBranchId = PrevEpochBranchId(consensusBranchId, consensus);
     uint256 dataToBeSigned;
     uint256 prevDataToBeSigned;
 
+    // Create signature hashes for shielded components.
     if (!tx.vJoinSplit.empty() ||
         !tx.vShieldedSpend.empty() ||
-        !tx.vShieldedOutput.empty())
+        !tx.vShieldedOutput.empty() ||
+        orchard_bundle.SpendsEnabled() ||
+        orchard_bundle.OutputsEnabled())
```

Comment:
Revert this change; we don't check Orchard signatures here, and it just causes us to unnecessarily call `SignatureHash` twice on pure-Orchard v5 transactions.